usdoj-crt/beta-ada
# Remove homepage image credits in footer from non-homepage pages

We only use images from Unsplash on the homepage. We only use images from Adobe stock images on our Topics pages. We want to make sure that the footer text aligns properly with the images being used. 

Changes:

1.  Added some conditional logic to the footer.html file to toggle between the different image attribution text depending on the page.

Test:

1.  Check that the attribution text is correct when on the homepage (Unsplash) and on any topics page (Adobe stock photos).
2.  Make sure that no attribution text appears on pages without any images.
